homeassistant/components/shelly/coordinator.py

```diff
@@ -145,9 +145,10 @@ def async_setup(self, pending_platforms: list[Platform] | None = None) -> None:
             name=self.name,
             connections={(CONNECTION_NETWORK_MAC, self.mac)},
             manufacturer="Shelly",
-            model=MODEL_NAMES.get(self.model, self.model),
+            model=MODEL_NAMES.get(self.model),
```

Comment:
```suggestion
            model=MODEL_NAMES.get(self.model, self.model),
```
Making sure we don't break this, thanks @gjohansson-ST

Reply:
https://github.com/home-assistant/core/pull/122040#discussion_r1680005347